certz/certz.proto

```diff
@@ -453,8 +453,7 @@ message Entity {
 
   oneof entity {
     Certificate certificate = 3;
-    CertificateBundle trust_bundle = 4;
```

Comment:
`trust_bundle` cannot be removed - it is used to rotate the chain of certificates that is used to validate certificate(s) used by the client when establishing the connection. It can be different than the one used by the server certificate that is sent to the client.

Reply:
ack, re-added in the next commit

Reply:
Thx for adding it back!

Looking at the definition of `CertificateChain` I realized that it is a better version of the `CertificateBundle`. Could you replace `CertificateBundle` with `CertificateChain` for the `trust_bundle` field? And remove the definition of `CartificateBundle`?

Reply:
Done in the most recent patch (https://github.com/openconfig/gnsi/pull/62/commits/4283316122401a71b5fac14f45e7201f7e93f8a0)